### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by jlvFlores**
During this pull request, I:
- Added hidden nav options
- Added a media query for desktop screens
- Updated tags on my headline section
- Rebuilt my recent works section in HTML
- Took almost no breaks 😪

After code review, I:
- Added a second media query for medium size screens
- Updated my background images
- Resolved an error with the navbar.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,125 +11,129 @@
...
-        <a href="#"><img src="assets/images/Logo Placeholder.png" alt="John Doe"></a>
-        <nav><i class="fas fa-bars"></i></nav>
+        <a href="#" class="logo"><img src="assets/images/Logo Placeholder.png" alt="John Doe"></a>
+        <nav>
+            <a href="#" class="nav-desktop">Portfolio</a>
+            <a href="#" class="nav-desktop">About</a>
+            <a href="#" class="nav-desktop">contact</a>
+            <img class="nav-desktop" src="assets/images/Icon - Mail.png" alt="mail icon">
+            <i class="fas fa-bars nav-mobile"></i>
+        </nav>
```
Comment (Shedrack-Sunday) [inline index.html]: - [x] Your Hamburger button still shows on the desktop view. kindly fix that 👍 .  A hamburger icon is typically used to represent a menu in a mobile view, as it takes up less space on a smaller screen. In a desktop view, there is typically more space available, so a menu can be displayed in a more traditional format, such as a list of links. Therefore, it is not necessary to show a hamburger icon in a desktop view. So kindly make the change 

# Your Design:
image

File: style.css [hunk 2]
```diff
@@ -399,3 +416,156 @@ footer a {
...
+  }
+
+  .headline {
+    display: grid;
+    height: 870px;
+    grid-template-columns: 140px 560px 1fr;
+    grid-template-rows: repeat(5, 1fr);
+    background-color: #1c1a19;
+    background-image: url('assets/images/illustration-header-mobile.png');
+    background-repeat: no-repeat;
```
Comment (Shedrack-Sunday) [inline style.css]: - [x] If you take a proper visual view of your design compared to that of the figma template, you'll notice that the background image on the figma design actually suspends from the top right corner, on the other hand, has a little bit of spacing which significantly makes a difference. The reason we emphasize sticking to this design is to get used to meeting project requirements, as you might end up IP building for a client either as a freelance or under a company. please make the c=necceary changes, Great job you did 👏 

# Yours:
image

# Figma :
image

File: style.css [hunk 3]
```diff
@@ -399,3 +416,156 @@ footer a {
   color: #dbd8d7;
   font-size: 19px;
 }
+
+@media screen and (min-width: 768px) {
+  .logo {
+    margin-left: 140px;
+    border: 1px solid #679;
+  }
```
Comment (Shedrack-Sunday) [inline style.css:425]: - [x] While I must commend the brilliant work you put in for this project, I suggest you adjust the styling starting at this media screen. from the screenshot below, you can see that the button overlaps with the text. seeing the awesome work you've done, please make the change and compliment the beautiful design 👏 

image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @jlvFlores     👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing most of the features from the project requirements , However, there are few you need to work on, to move to the next project. 💯 



Highlights
✅All linters are checked
✅ Descriptive commit messages 
✅ Mobile view still intact

Required Changes ♻️

- [x] Firstly, you did have a very beautiful design. however, the goal is to stick to the recommended one just as it will be in your career as a developer. I've made a detailed review of sections that needs fixing. kindly follow the suggestions, and you'll be approved in no time. Excellent work you did 👏 

_check the comment section for a detailed review_

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.


Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 4]
```diff
@@ -65,18 +84,18 @@ button:enabled:active {
...
   justify-content: center;
   background-color: #1c1a19;
-  background-image: url('assets/images/illustration-header-mobile.png');
+  background-image: url('assets/images/illustration-header.png');
   background-repeat: no-repeat;
-  background-position-x: calc(100% + 325px);
-  background-position-y: 118%;
+  background-position-x: calc(100% + 375px);
+  background-position-y: 35vh;
+  background-size: 85vh;
```
Comment (Tufahel) [inline style.css]: - [ ] Kindly fix the background image issue for the `heading.` I can't see any `background image` in the desktop version. It is a core requirement to follow the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=86lhUD7Kl0qMQAIy-0) as close as possible.
Screenshot from 2023-01-20 00-03-07

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @jlvFlores  :wave: 

Well done on implementing some of the required changes. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (jlvFlores) [reply style.css]: That is really odd. @Tufahel can you tell me the dimensions of your screen. I'm able to see the image in my laptop screen without any issues.

Comment (Tufahel) [reply style.css]: @jlvFlores, I am not sure what the issue is. But I can't see the background image. You have used it, and while inspecting, I can see that you have used it. [Video](https://www.loom.com/share/c7975fa1203b46df8b4dd558f2fa446d)

Screenshot from 2023-01-20 00-32-02

Comment (jlvFlores) [reply style.css]: @Tufahel Do you think this is a browser issue, or maybe you didn't pull the image assets? I'm not accusing, I'm just really stump as to why it doesn't show the background image on your screen despite working on mine and presumably on the previous CR's screen too. Could you kindly check if the other background images don't show either?

Comment (Tufahel) [reply style.css]: @jlvFlores, The issue is about the file name. You used `url('assets/images/illustration-header.png')` as `background image`. You need to use `url('assets/images/Illustration-header.png')`. The spelling of `illustration` was the issue. Kindly fix those lines below related to `background image`.

Screenshot from 2023-01-20 00-45-33

image

Comment (jlvFlores) [reply style.css]: Thank you so much, I'll fix it, and I'll be sure to not let it happen again.

**Review by BregornOriginal — APPROVED** [review]
## Hi @jlvFlores 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 👏🏼 👏🏼 👏🏼 

approve-applause

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work implementing changes requested by the previous reviewers 💯 💯 💯 

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**